types/minecraft-pinger/minecraft-pinger-tests.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2022 Northern Star
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
```

Comment:
```suggestion
```
optionally, unless you're under some obligation to have this copy left banner here

Reply:
Oh yeah Forgot that was there